src/services/impl/FileLogger.hh

```diff
@@ -21,7 +21,18 @@ class FileLogger : public ra::services::ILogger
 public:
     explicit FileLogger(const ra::services::IFileSystem& pFileSystem) noexcept
     {
-        m_pWriter = pFileSystem.AppendTextFile(pFileSystem.BaseDirectory() + L"RACache\\RALog.txt");
+        std::wstring sLogFilePath = pFileSystem.BaseDirectory() + L"RACache\\RALog.txt";
```

Comment:
Speed could be optimized more using `+=` or `append`. `L"RACache\\RALog.txt"` seems it can be a `constexpr `variable.
I was thinking of making something that attempts compile-time string concatenation. As far as I'm aware, none of the standard library does this.